migrations/20230203235010_create-bin-table.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema
+    .createTable("bin", (table) => {
+      table.increments();
+      table.text("color").notNullable();
+      table.text("name").notNullable();
```

Comment:
That’s looking ok, Please remember to AGD database file (sqlite) to gitignore 😊 do not include this file in PR or MR